bigflow/scaffold/scaffold.py

```diff
@@ -36,14 +36,4 @@ def start_project(
     )
 
     import bigflow.build.pip as bf_pip
-    bf_pip.pip_compile(project_path / "resources" / "requirements.txt")
-
-
-def migrate_project_from_10(
```

Comment:
What is motivation to remove this code?

Reply:
1. all projects inside allegro are already migrated
2. we are releasing version 1.3, I don't think we should keep migration tools forever
3. in connection to point 1, Im not sure if we have any users outside the allegro right now :(